core/src/StructureFactory.cpp

```diff
@@ -16,33 +16,29 @@
 
 #include "include/ParaGridIO.hpp"
 
+#include <ncAtt.h>
 #include <ncFile.h>
-#include <ncGroup.h>
-#include <ncGroupAtt.h>
 #include <stdexcept>
 #include <string>
 
 namespace Nextsim {
 
 std::string structureNameFromFile(const std::string& filePath)
 {
-    std::string structureName;
+    // std::string structureName;
+    std::string structureName = "parametric_rectangular";
 
-    try {
-        netCDF::NcFile ncf(filePath, netCDF::NcFile::read);
-        netCDF::NcGroup metaGroup(ncf.getGroup(IStructure::structureNodeName()));
-        netCDF::NcGroupAtt att = metaGroup.getAtt(IStructure::typeNodeName());
-        int len = att.getAttLength();
-        // Initialize a std::string of len, filled with zeros
-        structureName = std::string(len, '\0');
-        // &str[0] gives access to the buffer, guaranteed by C++11
-        att.getValues(&structureName[0]);
-        ncf.close();
-    } catch (const netCDF::exceptions::NcException& nce) {
-        std::string ncWhat(nce.what());
-        ncWhat += ": " + filePath;
-        throw std::runtime_error(ncWhat);
-    }
+    // FIXME: Temporary hack
+    // try {
+    //     netCDF::NcFile ncf(filePath, netCDF::NcFile::read);
+    //     netCDF::NcVar var = ncf.getVar("structure_name");
+    //     var.getVar(&structureName[0]);
+    //     ncf.close();
+    // } catch (const netCDF::exceptions::NcException& nce) {
+    //     std::string ncWhat(nce.what());
+    //     ncWhat += ": " + filePath;
+    //     throw std::runtime_error(ncWhat);
+    // }
```

Comment:
The temporary hack is to assume use of paragrid. Will need to figure out how to put the structure info back into the NetCDF files to get this logic working again.